Move secondary requirement texts out of the background (#2060)
act-rules/act-rules.github.io: _rules/link-non-empty-accessible-name-c487ae.md

    passed: further testing needed
    inapplicable: further testing needed
  wcag20:1.1.1: # Non-text content (A)

>     secondary: This success criterion is **related** to this rule. This happens for HTML `area` elements which are both links and as non-text content. Most failed examples satisfy this success criterion.

Reviewer: ```suggestion
    secondary: This success criterion is **related** to this rule. This is because HTML `area` elements are both links and non-text content. Most failed examples satisfy this success criterion.
```

>     secondary: This success criterion **partially overlaps** this rule. This is because the HTML`area` element count as both a link, and as non-text content. This is why some of the failed examples may not satisfy this success criterion.

Reviewer 2: ```suggestion
    secondary: This success criterion **partially overlaps** this rule. This is because the HTML `area` element counts as both a link and as non-text content. This is why some of the failed examples may not satisfy this success criterion.
```

>     secondary: This success criterion **partially overlaps** this rule. This is because the HTML `area` element counts as both a link and as non-text content. This is why some of the failed examples may not satisfy this success criterion.

Reviewer: ```suggestion
    secondary: This success criterion is **conditionally applicable** in this rule. It is applicable because the HTML `area` element counts as both a link and as non-text content. Most failed examples satisfy this success criterion.
```

Reviewer: I didn't see how an explanation would be written for [Keyboard Trap Example 7](https://w3c.github.io/wcag-act/act-rules-format.html#example-7be89ba3) in this PR, but I thought "partially" would work better there than here. 

For Ex.7: This SC is **partially covered** in this rule because the rule does not test for other possible solutions. As a result, this SC may be satisfied by some failed examples.
